pom.xml

```diff
@@ -52,17 +52,20 @@
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
     <jira-rest-client.version>5.2.7</jira-rest-client.version>
     <fugue.version>4.7.2</fugue.version>
-    <!-- jenkins -->
-    <jenkins.version>2.440.3</jenkins.version>
+    <!-- TODO: Remove when 2.472 is available in LTS -->
+    <jenkins.version>2.472</jenkins.version>
```

Comment:
We are supporting only LTS line, so let's make this a draft until there is an LTS released >=2.472
https://www.jenkins.io/changelog-stable/